README.md

```diff
@@ -53,6 +53,9 @@ Using Airflow to implement our ETL pipelines
     * production: `docker run --rm -p 80:8080 --name airflow  -v $(pwd)/dags:/usr/local/airflow/dags -v $(pwd)/service-account.json:/usr/local/airflow/service-account.json --env-file=./.env.production davidtnfsh/pycon_etl:prod webserver`
     * dev/test: `docker run --rm -p 80:8080 --name airflow  -v $(pwd)/dags:/usr/local/airflow/dags -v $(pwd)/service-account.json:/usr/local/airflow/service-account.json --env-file=./.env.staging davidtnfsh/pycon_etl:test webserver`
     * Note the difference are just the env file name and the image cache.
+4. Enter the `localhost:80` in the address bar in your browser. Open the Airflow Page.
```

Comment:
Is `:80` needed here? Web browsers default to the 80 port.

Reply:
Yes , not needed. Thank you.
I forget the default port is 80 for HTTP.